core/setup.py

```diff
@@ -23,7 +23,10 @@
     'cryptography==3.3.2',
     'pyyaml>=5.4.1, <6.0',
     'requests>=2.23.0, <3.0',
-    'Deprecated>=1.2.13, <1.3'
+    'Deprecated>=1.2.13, <1.3',
+    'opentelemetry-api>=1.6.2,<1.7',
+    'opentelemetry-sdk>=1.6.2,<1.7',
```

Comment:
looks like we can remove opentelemetry-api and -sdk as they seem to be transitive reps from exporter below.